app/styles/ember-arcgis-portal-components.scss

```diff
@@ -243,14 +241,18 @@ div.panel-body:has(.item-picker) {
         }
       }
       .item-picker-current-item-preview-meta {
+        display: flex;
+        flex-direction: column;
         font-size: 14px;
-        margin: 0px auto 15px auto;
+        margin-bottom: 15px;
         .glyphicon {
           margin-right: 5px;
         }
       }
     }
     .item-picker-controls {
+      display: flex;
+      flex-direction: column;
```

Comment:
We are using flexbox so it is _flexible_. We can kill the absolute positioning on this element (along with the `bottom` & `right` rules). This, in conjunction with the space-between rule above will push this to the bottom.